iOS/Flows/Conversation/ConversationViewControllerDataSource.swift

```diff
@@ -136,20 +136,17 @@ class ConversationCollectionViewDataSource: CollectionViewDataSource<Conversatio
         await Task.onMainActorAsync { [snapshot = snapshot, scrollToLatestMessage = scrollToLatestMessage] in
             self.apply(snapshot)
 
+            // Scroll to the latest message if needed and reconfigure cells as needed.
             if scrollToLatestMessage, let sectionIndex = snapshot.indexOfSection(sectionID) {
-                let latestMessageIndex: IndexPath
+                let latestMessageIndex = IndexPath(item: 0, section: sectionIndex)
```

Comment:
Simplified logic here. The latest message is always the first in the data source.